unit-tests/py/rspy/acroname.py

```diff
@@ -41,347 +41,344 @@ def usage():
     log.w( 'No acroname library is available!' )
     raise
 
-hub = None
-
-
 class NoneFoundError( RuntimeError ):
-    """
-    """
     def __init__( self, message = None ):
         super().__init__( self, message  or  'no Acroname module found' )
 
+class Acroname(usb_relay.usb_relay):
 
-def discover(retries = 0):
-    """
-    Return all Acroname module specs in a list. Raise NoneFoundError if one is not found!
-    """
+    hub = None
```

Comment:
If you need an instance member (looks like it, since you do `self.hub` elsewhere, then you need to initialize this in the constructor. The way it is now, it's a class (static) variable, not an instance variable.